### PR TITLE
Add htlm structure to list

### PR DESCRIPTION
**Pull request by CesarAlarcon-1983**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,9 +2,45 @@
...
+        <button> + </button>
+      </div>
+      <div class="clone">
+        <span> Clone </span>
+        <button> + </button>
+      </div>
+    </div>
+    <div class="list">
+      <header class="list-header">
+        <button class="newlistitem"> + </button>
```
Comment (migueljrojas) [inline index.html:27]: Add "-" dash to separate words in CSS class names

File: index.html [hunk 2]
```diff
@@ -2,9 +2,45 @@
...
-    <h1>"Hello World"</h1>
-  </body> 
+    <h1 class="titulo"> To-Do List </h1>
+    <div class="options">
+      <div class="newlist">
+        <span> Add New List </span>
+        <button> + </button>
+      </div>
+      <div class="clone">
+        <span> Clone </span>
```
Comment (migueljrojas) [inline index.html:21]: Watch out white spaces

File: index.html [hunk 3]
```diff
@@ -2,9 +2,45 @@
 <html>
   <head>
     <meta charset="utf-8">
-    <title></title>
+    <link
```
Comment (migueljrojas) [inline index.html:5]: no need for placing attributes in new lines

**Review by migueljrojas — CHANGES_REQUESTED** [review]

File: index.html (hunk 3 above)
Comment (CesarAlarcon-1983) [reply index.html:5]: Corrected

File: index.html (hunk 2 above)
Comment (CesarAlarcon-1983) [reply index.html:21]: Corrected in all lines

File: index.html (hunk 1 above)
Comment (CesarAlarcon-1983) [reply index.html:27]: Corrected in all lines

**Pull request merged**